src/ui/components/panels/loadout/active_loadout/CCCombatDialog.vue

```diff
@@ -121,7 +121,7 @@ export default Vue.extend({
         this.displayLog = this.action.AnyUsed
         // eslint-disable-next-line @typescript-eslint/no-this-alias
         const self = this
-        this.$emit('use')
+        this.$emit('use', free)
```

Comment:
This was previously a broken link in the "use" emitter chain; `free` will now continue to bubble up correctly.